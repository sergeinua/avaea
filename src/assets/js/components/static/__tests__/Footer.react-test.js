import React from 'react';
import StaticFooter from '../Footer';
import renderer from 'react-test-renderer';

test('Footer component test', () => {
  var mockUser = {email: "test@gmail.com", id: 58};
  var mockLocation = {pathname: '/home'};
  var component = renderer.create(
    <StaticFooter user={mockUser} location={mockLocation}/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
