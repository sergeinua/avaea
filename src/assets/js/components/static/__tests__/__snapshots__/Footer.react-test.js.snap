exports[`test Footer component test 1`] = `
<footer
  className="static">
  <div
    className="top">
    <div
      className="wrapper">
      <div
        className="left">
        <nav>
          <ul
            className="menu nav">
            <li
              className="">
              <a
                onClick={[Function]}
                style={Object {}}>
                About
              </a>
            </li>
            <li
              className="">
              <a
                onClick={[Function]}
                style={Object {}}>
                Team
              </a>
            </li>
            <li
              className="">
              <a
                onClick={[Function]}
                style={Object {}}>
                Contact
              </a>
            </li>
          </ul>
        </nav>
        <div
          className="holder">
          <ul
            className="terms">
            <li>
              <a
                onClick={[Function]}
                style={Object {}}>
                Terms and Conditions
              </a>
            </li>
            <li>
              <a
                onClick={[Function]}
                style={Object {}}>
                Privacy
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        className="right">
        <a
          className="social fb"
          href="http://www.facebook.com/onvoya"
          target="_blank" />
        <a
          className="social twitter"
          href="http://www.twitter.com/onvoya"
          target="_blank" />
        <a
          className="social instagram"
          href="http://www.instagram.com/onvoya"
          target="_blank" />
      </div>
    </div>
  </div>
  <div
    className="bottom">
    <span
      className="copyright">
       2017 OnVoya.com
    </span>
  </div>
</footer>
`;
